README.md

```diff
@@ -32,10 +32,13 @@ containing:
 ```python
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
+# rules_nodejs provides build rules related to NPM and Yarn package management.
+# These rules are used for modeling the dependencies between user TypeScript
+# code and NPM or Yarn packages.
 git_repository(
     name = "build_bazel_rules_nodejs",
     remote = "https://github.com/bazelbuild/rules_nodejs.git",
-    tag = "0.0.2", # check for the latest tag when you install
+    tag = "0.3.1", # check for the latest tag when you install
```

Comment:
I'd clarify here too: where to check (i.e. the bazelbuild/rules_typescript/WORKSPACE file correct? This has to match what's in this project's WORKSPACE as I understand?)

Reply:
I believe the current instructions mean to pick the latest tag from https://github.com/bazelbuild/rules_nodejs/releases, irrespective of what's in rules_typescript's WORKSPACE. I wasn't under the impression the versions of these repos need to match across workspaces.

I've clarified the comment to say where to check for the latest tag. Can one of the admins verify my interpretation?

Reply:
it's possible for the latest rules_nodejs to contain a breaking change, where version x of rules_typescript doesn't work with it. However, the latest version of rules_typescript *ought* to always work with rules_nodejs. Sadly WORKSPACE doesn't have the peerDependency or dependency range semantics of npm that allow rules_typescript to indicate the range of versions of rules_nodejs that work.

For this reason, I plan to remove the versioning of rules from WORKSPACE to the extent possible, where a package.json could be used instead.

I think renovate is the answer here - if I install it on this repo, it will keep the version of rules_nodejs up-to-date in the WORKSPACE, along with CI that ensures it works. Then the correct guidance is to copy the rules_nodejs version from the rules_typescript WORKSPACE (manually emulating the missing transitive dependencies in Bazel)